app/settings.py

```diff
@@ -146,6 +146,14 @@ def utcoffset_or_fail(date_value, key):
 
 SDS_API_BASE_URL = os.getenv("SDS_API_BASE_URL")
 
+OIDC_TOKEN_BACKEND = os.getenv("OIDC_TOKEN_BACKEND")
+
+OIDC_TOKEN_LEEWAY_IN_SECONDS = int(os.getenv("OIDC_TOKEN_LEEWAY_IN_SECONDS", "300"))
+
+SDS_OAUTH2_CLIENT_ID = os.getenv(
+    "SDS_OAUTH2_CLIENT_ID", "test-client-id.apps.googleusercontent.com"
```

Comment:
Test client id shouldn't be committed. Defaults should be prod first.

Reply:
Good point, should i remove the default entirely for now then since we won't know prod default?

Reply:
Yes. We won't expose it in the public source anyway even if it is not strictly sensitive. We will pass it in when deploying the app.